src/modules/privacy-idea-administration/privacy-idea-administration.module.ts

```diff
@@ -1,11 +1,13 @@
-import { Module } from '@nestjs/common';
-import { PrivacyIdeaAdministrationService } from './privacy-idea-administration.service.js';
-import { PrivacyIdeaAdministrationController } from './privacy-idea-administration.controller.js';
 import { HttpModule } from '@nestjs/axios';
+import { Module } from '@nestjs/common';
 import { PersonModule } from '../person/person.module.js';
+import { PersonenKontextModule } from '../personenkontext/personenkontext.module.js';
+import { PrivacyIdeaAdministrationController } from './privacy-idea-administration.controller.js';
+import { PrivacyIdeaAdministrationService } from './privacy-idea-administration.service.js';
+import { ServiceProviderApiModule } from '../service-provider/service-provider-api.module.js';
 
 @Module({
-    imports: [HttpModule, PersonModule],
+    imports: [HttpModule, PersonModule, PersonenKontextModule, ServiceProviderApiModule],
```

Comment:
Should import `ServiceProviderModule` instead of `ServiceProviderApiModule`

Reply:
Fixed in [5596065](https://github.com/dBildungsplattform/dbildungs-iam-server/pull/648/commits/55960656bfa01f7dbb6cd2368742605f7e14b81c)